pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/service/KitCheckService.java

```diff
@@ -124,7 +124,7 @@ public KitCheckResult processPotentialKitRecipient(String studyGuid, long kitTyp
 
         if (candidate.getAddressValidationStatus() == null
                 || candidate.getAddressValidationStatus() == DSM_INVALID_ADDRESS_STATUS) {
-            log.warn("Participant {} has an invalid mailing address", userGuid);
+            log.debug("Participant {} has an invalid mailing address", userGuid);
```

Comment:
Rare cases , investigating in prod why kit wasnt created, these statements used to help that ptp was skipped. 
ok to change though to reduce noise because Housekeeping keeps checking and logging same